microprofile/WFLY-19866-promote-mp-rest-client-to-default.adoc

```diff
@@ -0,0 +1,156 @@
+---
+categories:
+- microprofile
+- jaxrs
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/640
+feature-team:
+ developer: jamezp
+ sme:
+  - jamezp
+ outside-perspective:
+  - TBD
```

Comment:
Is this something that needs to be done before merge of this PR?

Reply:
I think so, yes. I'll double check that though.

Reply:
Thank you! I believe that this thread can be resolved now.